packages/pigeon/README.md

```diff
@@ -91,9 +91,9 @@ Nested datatypes are supported, too.
 
 ### Asynchronous Handlers
 
-By default Pigeon will generate synchronous handlers for messages.  If you want
-to be able to respond to a message asynchronously you can use the `@async`
-annotation as of version 0.1.20.
+By default Pigeon will generate synchronous handlers for messages on the native side. 
+The Flutter part is not asynchronous! If you want to be able to respond to 
+a message asynchronously you can use the `@async` annotation as of version 0.1.20.
```

Comment:
What is generated on the native side depends on if you have a `@HostApi` or `@FlutterApi`.

Using an exclamation isn't really the right tone for the document =)

Reply:
Thank you for your reply. I totally agree. Just wanted to point it out and I was not sure what is the best way.
Should I try to come up with better wording or can you assign it to somebody to rephrase these two parts ? :)

Reply:
It's you, I believe in you, you got this.  You ran into something that frustrates you in the documentation.  You have all the information on how to make this better.  Make it better, make it accurate, make it professional and it's in =)

Reply:
Ok hopefully this version will be more understandable for users :) 

P.S.: So glad to start giving back to the community after developing in Flutter for 3 years.